src/helpers.ts

```diff
@@ -359,3 +359,71 @@ export function courseNumAsDecimal(courseNumber: string) {
         return parseInt(secondToLastElement);
     }
 }
+
+export function HexToHSL(hex: string): { h: number; s: number; l: number } {
```

Comment:
Are you still planning to implement the thing with changing the hue/lightness of the colors for discussions/labs? If not, can you remove these functions?

Reply:
Reminder of the discussion results from the weekly meeting: 
1. open a new issue about making labs and discussions different hues from the lecture
2. copy these functions to a comment on that issue
3. remove them from this feature branch
4. re-request review.